Core/ConfigWizard.cs

```diff
@@ -46,12 +46,12 @@ public static void RunConfigWizard(bool useDefault)
             return;
         }
 
-        LocalAdmin.Configuration.RestartOnCrash = BoolInput("Should the server be automatically restarted after a crash?");
-        LocalAdmin.Configuration.LaLiveViewUseUtc = !BoolInput("Should timestamps in the LocalAdmin live view use server timezone (otherwise UTC time will be use)?");
+        LocalAdmin.Configuration.RestartOnCrash = BoolInput("Should the server automatically restart after a crash?");
```

Comment:
Invalid suggestion. The game server won't restart itself. It will be restarted by the LocalAdmin.